packages/terra-tabs/src/TabUtils.jsx

```diff
@@ -0,0 +1,33 @@
+const KEYCODES = {
+  ENTER: 13,
+  SPACE: 32,
+  LEFT_ARROW: 37,
+  RIGHT_ARROW: 39,
+};
+
+/**
+ * Returns if the selected tab is not the currently selected index.
+ */
+const shouldHandleSelection = (currentSelectedKey, newSelectedKey) => newSelectedKey !== currentSelectedKey;
+
+/**
+ * Returns the active key if given, else returns the key of the first tab.
+ * To be used in the constructor, to set initial state.
+ */
+const initialSelectedTabKey = (tabs, activeKey) => {
+  if (activeKey) {
+    return activeKey;
+  } else if (tabs.length) {
+    return tabs[0].key;
+  }
+  return tabs.key;
```

Comment:
Is the last `return` statement needed? It seems like the first 2 conditionals handle the use cases described in the code comment?

Reply:
If there is only 1 tab, then this would be needed. Children only become an array when there are 2 or more children. When there is 1 child it's of type element and doesn't have the length property and the key should be accessed directly.

Reply:
:+1: